polyply/data/martini3/P3HT.martini3.ff

```diff
@@ -6,12 +6,12 @@ P3HT               1
 ;
 [ atoms ]
 ;  nr  type  resnr residue  atom  cgnr    charge   mass
-    1   TC6    1    P3HT     S1    1        0      
-    2   TC5    1    P3HT     C2    2        0      
-    3   TC5    1    P3HT     C3    3        0      
+    1   TC6    1    P3HT     S1    1        0        36
+    2   TC5    1    P3HT     C2    2        0        36
+    3   TC5    1    P3HT     C3    3        0        36
     4     U    1    P3HT     V4    4        0        0  
-    5   SC2    1    P3HT     C5    5        0     
-    6   SC2    1    P3HT     C6    6        0     
+    5   SC2    1    P3HT     C5    5        0        36
+    6   SC2    1    P3HT     C6    6        0        36
```

Comment:
mass should be 54 beads number 5 and 6

Reply:
Ahh I might have missed this because the reference itp also has 36. Not sure where this comes from but I've fixed it.